app/script/view_model/content/PreferencesOptionsViewModel.js

```diff
@@ -81,8 +83,10 @@ z.viewModel.content.PreferencesOptionsViewModel = class PreferencesOptionsViewMo
     }
 
     amplify.publish(z.event.WebApp.WARNING.MODAL, z.viewModel.ModalsViewModel.TYPE.ACKNOWLEDGE, {
-      message: z.l10n.text(z.string.modalCallEmptyLogMessage),
-      title: z.l10n.text(z.string.modalCallEmptyLogHeadline),
+      text: {
```

Comment:
I am kind of surprised that the payload changed, but nothing changed on the consumer's side. 
Any clue on this?

Reply:
I just made a mistake the first time around. This is the proper way to use our warning modals.